README.md

```diff
@@ -82,11 +82,25 @@ people shouldn't have indefinite access to, like a share link.
 => nil
 
 >> explicit_expiring_sgid = SecretAgentMessage.find(5).to_sgid(expires_at: Time.now.advance(hours: 1))
-=> #<SignedGlobalID:0x008fde45df8937
+=> #<SignedGlobalID:0x008fde45df8937 ...>
 
 # 1 hour later...
 >> GlobalID::Locator.locate_signed explicit_expiring_sgid
 => nil
+
+>> never_expiring_sgid = Document.find(5).to_sgid(expires_in: nil)
+=> #<SignedGlobalID:0x008fde45df8937 ...>
```

Comment:
Would break off the code examples just before here to say:

> Passing a falsey value in either `expires_in` or `expires_at` turns off expiration entirely.

>  ```ruby
>  >> never_expiring...
> ```